### PR TITLE
DEV TO MAIN

### PR DESCRIPTION
**Pull request by SukrutN**
DEV TO MAIN

### CODE REVIEW

File: public/images/team/team-01.png
Comment (ThatZiv) [inline public/images/team/team-01.png:1]: Who's this handsum fella

**Review by XLRA — APPROVED** [review]
Looks good to me! 

Congratulations

**Review by ThatZiv — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
+# Trophy Tracker
+
+![banner](https://trophy-tracker-wsu.vercel.app/_next/image?url=%2Fimages%2Flogo%2Flogo-white.png&w=256&q=75)
```
Comment (PierreT20) [inline README.md:3]: LGTM

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -33,6 +33,8 @@ jobs:
         run: npm install
       - name: Lint
         run: npm run lint
+      - name: Test
```
Comment (PierreT20) [inline .github/workflows/deploy.yml:36]: Impressive!

**Review by PierreT20 — APPROVED** [review]

File: public/images/team/team-06.png
Comment (ThatZiv) [inline public/images/team/team-06.png:1]: Is this a stock image? I'm not exactly sure of the origins of this image :|

Comment (XLRA) [inline public/images/team/team-06.png:1]: I feel like I saw this guy somewhere... Not sure where though.

**Pull request merged**